src/app/pages/quickMaps/pages/projection/download/download.component.scss

```diff
@@ -0,0 +1,62 @@
+.container-row {
+    // display: grid;
+    // grid-template-rows: 1fr 1fr 1fr;
+    // grid-template-columns: auto;
+    // grid-template-areas: "png svg csv";
+    display: flex;
+    flex-flow: row wrap;
+    align-items: flex-start;
+    justify-content: flex-start;
```

Comment:
I'd suggest making this `center`

also adding to this class:
`text-align: center`